crates/omnix-cli/crate.nix

```diff
@@ -32,9 +33,28 @@ in
         pkgsStatic.openssl
       ];
 
-      # Disable tests due to sandboxing issues; we run them on CI
-      # instead.
-      doCheck = false;
+      cargoTestExtraArgs = "-- " + (lib.concatStringsSep " " [
+        # requires networking
+        "--skip=command::ci::build_flake_output"
+        "--skip=command::ci::test_haskell_multi_nix"
+        "--skip=command::ci::test_haskell_multi_nix_all_dependencies"
+        "--skip=command::ci::test_services_flake"
+        "--skip=command::show::om_show_nixos_configurations"
+        "--skip=command::show::om_show_remote"
+
+        # required health checks, like "Flakes Enabled" and "Max Jobs" fails in sandbox
+        "--skip=command::health::om_health"
+
+        # error: creating directory '/nix/var/nix/profiles': Permission denied
+        # TODO: elaborate on the error
+        "--skip=command::init::om_init"
+
+        # FIXME: This should pass in the sandbox
+        # > ❄\u{fe0f}  nix -j auto eval --json --override-input flake ../.. --override-input flake-schemas /nix/store/0npxvlidr0w1b99lrvcmd2wd46y2kibh-flake-schemas --override-input systems /nix/store/g1pv78p6lk92hjzr7syrcihvj4rx1fnz-source --no-write-lock-file \'/nix/store/qm8q4kylp1jy68k34c2bskpamxgyc9ix-source#contents.excludingOutputPaths\' --quiet --quiet\u{fe0f}
+        # > error: experimental Nix feature \'nix-command\' is disabled; use \'--extra-experimental-features nix-command\' to override
+        # > Error: Unable to fetch flake
+        "--skip=command::show::om_show_local"
+      ]);
```

Comment:
@srid This could be a potential bug. I even tried looking at how `om ci` works, but even that uses the same `NixCmd::Default`, so it is likely that even `om ci` doesn’t automatically apply the `experimental-features` when not enabled by default.

We don’t have a similar `om_ci_local` test (that can run in Nix sandbox), which could be why it went undetected.